### DISCUSSION TITLE
Grid card: Set title via the visual editor in UI

### DISCUSSION THREAD

**Discussion by awolkers** [home-assistant/frontend#18123] [Dashboard] [answered]
Currently you can only set the title of a grid card if you change to editing the card in yaml mode. Would be nice to have an option to add a title via the visible editor as well.

**Comment by Kanecaine** [comment]
agree - and would love to see this on vertical and horizontal stack too.

**Comment by KNXBroker** [comment]
Also agree, even better: an official title card as available for the mushroom cards. Being able to link another page is great.

**Comment by karwosts** [comment, marked as answer]
Title was added to UI for grid back in january, and will be added for h/v stack in the upcoming release. 

Closing.
